README.md

```diff
@@ -8,18 +8,18 @@ Percona Distribution for MongoDB is a collection of open-source solutions to run
 
 Percona Distribution for MongoDB consists of the following components:
 
-- [Percona Server for MongoDB](https://www.percona.com/doc/percona-server-for-mongodb/4.4/index.html) - a fully-compatible open-source replacement for MongoDB Community Edition.
+- [Percona Server for MongoDB](https://www.percona.com/doc/percona-server-for-mongodb/4.2/index.html) - a fully-compatible open-source replacement for MongoDB Community Edition.
```

Comment:
The URL should include 5.0 here.Please kindly update it

Reply:
Updated